bindings/pyroot/pythonizations/CMakeLists.txt

```
@@ -154,7 +154,7 @@ add_library(${libname} SHARED ${cpp_sources})
 add_dependencies(${libname} ROOTPythonizationsPySources)
 
 # Set the suffix to '.so' and the prefix to 'lib'
-set_target_properties(${libname} PROPERTIES  ${ROOT_LIBRARY_PROPERTIES})
+set_target_properties(${libname} PROPERTIES ${ROOT_LIBRARY_PROPERTIES_NOVER})
```

```suggestion
set_target_properties(${libname} PROPERTIES ${ROOT_LIBRARY_PROPERTIES_NO_VERSION})
```
I would suggest a more explicit name. Just from reading `NOVER`, I don't know that it is.